wellcomecollection/terraform-aws-sqs
# format and release: out with BK, in with GHA

## What does this change?

Move the format and release workflows to GHA to align with the other TF modules
BK pipeline infra removed https://github.com/wellcomecollection/buildkite-infrastructure/pull/29

## How to test

On push to any branch but main, the format workflow should run in GHA.
BK pipeline should not run 

## How can we measure success?

GHA reduce the need for extra scripts and docker stuff
This contributes to our platform health OKR

> Reduce lines of code associated with our Continuous Integration & Delivery services by prioritising migration to GitHub actions and removing unnecessary build scripts. 

## Have we considered potential risks?

The release workflow has not been tested e2e in the wild. However it mirrors that of `scala-libs` which is working fine
